radionets/evaluation/jet_angle.py

```diff
@@ -111,8 +111,8 @@ def calc_jet_angle(image):
             maxima.extend([a])
 
     vals = torch.tensor(maxima)
-    x_mid = vals[:, 0]
-    y_mid = vals[:, 1]
+    x_mid = 32
```

Comment:
this can lead to problems when sources don't have their maximum in the center of the image (future simulations)
but then we will face more problems with our functions, I guess

Reply:
I can revert the changes. I agree that this is a more general solution, but it is sometimes buggy.